core/identity/src/autoconf.rs

```diff
@@ -0,0 +1,37 @@
+use std::env;
+
+use ethsign::*;
+use rustc_hex::FromHex;
+
+use ya_core_model::NodeId;
+
+use crate::id_key::IdentityKey;
+use anyhow::Context;
+
+// autoconfiguration
+const ENV_AUTOCONF_PK: &str = "YAGNA_AC_IDENTITY_PK";
+const ENV_AUTOCONF_APP_KEY: &str = "YAGNA_AC_APPKEY";
```

Comment:
```suggestion
const ENV_AUTOCONF_APP_KEY: &str = "YAGNA_AUTOCONF_APP_KEY";
```

Reply:
Requestors already use `YAGNA_APPKEY`. naming new variable to `YAGNA_AUTOCONF_APP_KEY` would be incosistent.